integrations/gitlab/src/webhooks.ts

```diff
@@ -171,6 +194,10 @@ export async function handleMergeRequestEvent(
                         standalone: {
                             ref: sourceRef,
                         },
+                        eventCreatedAt:
```

Comment:
Same here, I don't think that it's necessary for merge requests events

Reply:
Yep!